frontend/src/pages/NewRun.tsx

```diff
@@ -272,19 +294,20 @@ class NewRun extends Page<{}, NewRunState> {
             </React.Fragment>
           )}
 
+          {/* Run parameters form */}
           <div className={commonCss.header}>Run parameters</div>
           <div>{this._runParametersMessage(pipeline)}</div>
-
           {pipeline && Array.isArray(pipeline.parameters) && !!pipeline.parameters.length && (
             <div>
               {pipeline.parameters.map((param, i) =>
                 <TextField id={`newRunPipelineParam${i}`} key={i} variant='outlined'
                   label={param.name} value={param.value || ''}
                   onChange={(ev) => this._handleParamChange(i, ev.target.value || '')}
-                  style={{ height: 40, maxWidth: 600 }} className={commonCss.textField} />)}
+                  style={{ maxWidth: 600 }} className={commonCss.textField}/>)}
```

Comment:
Why remove height? What happens when you have long parameter names?

Reply:
Height here is a bit of a misnomer here. I've actually go a follow-up that uses `InputProps` to change what you'd expect `height` to change. This change is essentially a noop in this case. Extremely long parameter names go off the end of the text box, but I don't think this is something we'll have to worry about